FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -68,5 +67,58 @@ public void ExecuteTestInvalidCredentials()
             FireboltException exception = Assert.Throws<FireboltException>(() => conn.Open());
             Assert.IsTrue(exception.Message.Contains("403") || exception.Message.Contains("429"));
         }
+
+        [Test]
+        public void ExecuteSelectTimestampNtz()
+        {
+            var connString =
+                $"database={Database};username={Username};password={Password};endpoint={Endpoint};account={Account}";
+
+            using var conn = new FireboltConnection(connString);
+            conn.Open();
+            conn.SetEngine(Engine);
+            var command = conn.CreateCursor();
+            command.Execute("SELECT '2022-05-10 23:01:02.0'::timestampntz");
+            DateTime dt = new DateTime(2022, 5, 10, 23, 1, 2, 0);
```

Comment:
What about the microsecond precision of timestamp(n)tz? We should check that .net sdk can deal with those without losing precision.

Reply:
milliseconds for timestamptz and timestampntz are already tested in [FireboltCommandTest.cs](https://github.com/firebolt-db/firebolt-net-sdk/pull/12/files#diff-8909c06177d5929e9f73c9361e4064bafeb1d53685c2db88eb64149c97238666) - I didn't want to copy/paste the exact same scenario

Reply:
I still think it's worth testing the full precision, since the other one is a unit test. It can be a different value but in an integration test it would verify everything works end-to-end.
Also in a linked test precision is lost? https://github.com/firebolt-db/firebolt-net-sdk/pull/12/files#diff-8909c06177d5929e9f73c9361e4064bafeb1d53685c2db88eb64149c97238666R224

Reply:
It's added at the next lime as the constructor only permits to specify milliseconds

Reply:
Ah, missed that part. Fair enough.

Reply:
> I still think it's worth testing the full precision, since the other one is a unit test. It can be a different value but in an integration test it would verify everything works end-to-end. 
Just added them 😄